samtranslator/model/eventsources/push.py

```diff
@@ -788,6 +788,35 @@ def _add_swagger_integration(self, api, function, intrinsics_resolver):
                     path=self.Path, method_name=self.Method, request_model=self.RequestModel
                 )
 
+                validate_body = self.RequestModel.get("ValidateBody")
```

Comment:
You may still need to do a `None` check but if you default this to `false`. I can't think if there is a way to get `None` if you default (probably because it is late). Though you should be able to prove that through testing. Could simply the below logic.

You can test for both false then and not add any validators. Maybe that is confusing if you want to audit the OpenApi Definition later and the validators disappear even if they were both set to `false`. So maybe what you have is better anyways. Sorry thinking while I type, but leaving it to strike discussion.

Reply:
Yes. My first thought is when we want explicitly False. If is not defined I am treating it as a None, but if it is False, then I would consider it as a defined field and even if it is False *I think* we should set it, as it will be explicitly on the configs saying we don't want validate anything.